src/libstd/path.rs

```diff
@@ -1647,7 +1647,7 @@ impl From<&Path> for Arc<Path> {
 
 #[stable(feature = "shared_from_slice2", since = "1.24.0")]
 impl From<PathBuf> for Rc<Path> {
-    /// Converts a Path into a Rc by copying the Path data into a new Rc buffer.
+    /// Converts a PathBuf into a Rc by moving the PathBuf data into a new Rc buffer.
```

Comment:
```suggestion
    /// Converts a `PathBuf` into an `Rc` by moving the `PathBuf` data into a new `Rc` buffer.
```

Reply:
It would be good to put backticks around the comment in `impl From<&Path> for Rc<Path>` too.